### DISCUSSION TITLE
How can I handle schemaVersion during development ?

### DISCUSSION THREAD

**Discussion by sprhawk** [simolus3/drift#2795] [General]
To say during development, I bumped  my schemaVersion to 5, and I added one column, write some migration to migrate from 4 to 5, then run the app, drift will upgrade my database.  I find I need to add another column or other changes to my database, if I keep the schemaVersion to 5, when I override my current installment, when I started my app, the drift will not call the migration. Should I bump the schemaVersion to 6, or other techniques should I use to handle this ?

Thanks

**Comment by simolus3** [comment]
Strictly speaking, you don't need to increase the schema version until you publish a release to users. That version bump can then include all the changes since the last version as a single schema migration.

But as long as you're not incrementing the schema version getter, drift also won't call your migrator. The only alternative during development is to delete the data of your app (or at [least the database](https://drift.simonbinder.eu/docs/migrations/#during-development)) after making changes without writing a migration.

**Reply by sprhawk** [reply]
OK, thanks
